src/Microsoft.OData.Client/DeepInsertSaveResult.cs

```diff
@@ -112,6 +142,7 @@ internal void BuildDescriptorGraph<T>(IEnumerable<Descriptor> descriptors, bool
                         throw Error.InvalidOperation(Strings.Context_DeepInsertOneTopLevelEntity);
                     }
 
+                    topLevelDescriptor.ContentGeneratedForSave = true;

```

Comment:
what does this do?

Reply:
Ensure when we process an entity descriptor, we also process the related links. see https://github.com/OData/odata.net/blob/4414000c0da72b22ff3c4a96f51cbf35f2316bca/src/Microsoft.OData.Client/BaseSaveResult.cs#L836

Reply:
`ContentGeneratedForSave` means that the descriptor should be processed during `SaveChanges`. It's an existing property that we are re-using.